api/company-user/index.js

```diff
@@ -41,6 +41,32 @@ exports.activationStatus = async (ctx) => {
   };
 };
 
+exports.changePassword = async (ctx) => {
+  const {
+    newPassword, currentPassword
+  } = ctx.request.body.passwords;
+
+  let { user } = ctx;
+
+  const PasswordMatchesCurrent = await user.compareHash(currentPassword);
+
+  if (!PasswordMatchesCurrent) {
```

Comment:
```suggestion
  if (!await user.compareHash(currentPassword)) {
```
No need for this var in async land.

Reply:
good call